base/exports.jl

```diff
@@ -1149,6 +1149,9 @@ public
     isoperator,
     isunaryoperator,
 
+# scalar math
+    uabs,
```

Comment:
Is there really no existing category to put this in?

Reply:
this exact category already exists in the export list above. so I think it's fine to use the same categories for the public list here as well

Reply:
I'm adding `uabs` to the `public` list in `base/exports.jl`. Maybe it would be clearer if this list was in a different file. Or I could add `public uabs` right before the main docstring of `uabs` instead?

Reply:
It does seem like maybe we should move the `public` list to a separate file named `base/public.jl`, to reduce confusion?

Reply:
https://github.com/JuliaLang/julia/pull/53487